common/alignment.py

```diff
@@ -951,13 +951,13 @@ def calculate_statistics(self, ignore={}):
                 for segment, segment_num in self.aa_count.items():
                     self.amino_acid_stats[i].append([])
                     k = 0
-                    if segment == 'Custom' and segment_num == 'x':
-                        sorted_res = sorted(segment_num, key=lambda x: (x.split("x")[0], x.split("x")[1]))
-                    elif segment == 'Custom' and segment_num == '.':
-                        sorted_res = sorted(segment_num, key=lambda x: (x.split(".")[0], x.split(".")[1]))
-                    else:
-                        sorted_res = sorted(segment_num)
-                    for gn in sorted_res:
+                    # if segment == 'Custom' and segment_num == 'x':
+                    #     sorted_res = sorted(segment_num, key=lambda x: (x.split("x")[0], x.split("x")[1]))
+                    # elif segment == 'Custom' and segment_num == '.':
+                    #     sorted_res = sorted(segment_num, key=lambda x: (x.split(".")[0], x.split(".")[1]))
+                    # else:
+                    #sorted_res = segment_num
+                    for gn in self.generic_numbers[self.numbering_schemes[0][0]][segment]:
```

Comment:
I think this is fine but just to double-check. GPCRdb Class A numbering scheme is used here throughout (that's why the nested 0 index call), so no matter what class sequence is aligned it is comparable, right?

Reply:
I tested this in multiple situations and independent of the situation, the 0 0 index always gave me the main numbering scheme that is being used internally in the alignment module. To not make it static and to allow for changes in the rest of the code I decided to use the index instead of just hardcoding it.